src/controller_examples/zookeeper_controller/proof/liveness/liveness_property.rs

```diff
@@ -27,7 +27,7 @@ use crate::kubernetes_cluster::{
 use crate::temporal_logic::{defs::*, rules::*};
 use crate::zookeeper_controller::{
     common::*,
-    proof::{common::*, safety, terminate},
+    proof::{common::*, liveness::invariants as specific_invariants, liveness::terminate},
```

Comment:
Maybe name invariants.rs to helper_invariants.rs?

Reply:
Done.